.travis.yml

```diff
@@ -26,5 +26,8 @@ script:
     - cd ..
     - ./build/merlict-test
 
-
+    - echo "trying out one_source.py"
```

Comment:
Very nice, this is the only way to make sure the one-source-feature is maintained in the future